biz.aQute.resolve/test/biz/aQute/resolve/obr/plugin/ObrExporterTest.java

```diff
@@ -0,0 +1,149 @@
+package biz.aQute.resolve.obr.plugin;
+
+import static aQute.bnd.build.Workspace.findWorkspace;
+import static aQute.lib.io.IO.getFile;
+import static biz.aQute.resolve.Bndrun.createBndrun;
+import static biz.aQute.resolve.obr.plugin.ObrExporter.TYPE;
+import static java.util.zip.GZIPInputStream.GZIP_MAGIC;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import aQute.bnd.osgi.repository.XMLResourceParser;
+import aQute.lib.io.IO;
+import biz.aQute.resolve.Bndrun;
+
+public final class ObrExporterTest {
+
+    private Bndrun      project;
+    private ObrExporter	underTest;
+
+    @Before
+    public void setup() throws Exception {
+        project = createBndrun(findWorkspace(IO.work), getFile("testdata/obr-exporter/test.bndrun"));
+        underTest = new ObrExporter();
+    }
+
+    @After
+    public void teardown() throws IOException {
+        final File testResource1 = IO.getFile("testdata/obr-exporter/target/test.xml");
```

Comment:
This shows the problem with using the real Bnd workspace for testing. You make files in the real workspace which must be cleaned up. This should not be necessary since your test workspace should be in an output folder.

Reply:
DONE ✔️